### PR TITLE
Change 1.14 to 1.15 in SETUP.md

### PR DESCRIPTION
**Pull request by dominikaaaa**

### CODE REVIEW

**Comment by S-B99** [comment]
Bump

File: SETUP.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 The easiest way to install Baritone is to install [Impact](https://impactclient.net/), which comes with Baritone.
 
-For 1.14.4, [click here](https://www.dropbox.com/s/rkml3hjokd3qv0m/1.14.4-Baritone.zip?dl=1).
+1.14.4 and lower is located on the [releases](https://github.com/cabaletta/baritone/releases/) page, if you don't want to use Impact.
```
Comment (0-x-2-2) [inline SETUP.md]: Just add the links for 1.13, 1.14 and 1.15 versions then tell them to go to releases for forge 1.12???

Comment (S-B99) [reply SETUP.md]: What? That is a link to them

Comment (S-B99) [reply SETUP.md]: If I link directly to the release versions this file will have to be changed *every* time there is a new release, not just for major releases.

Comment (CDAGaming) [reply SETUP.md]: Plus, isn't 1.15.2 ALSO on the releases page xD

Comment (S-B99) [reply SETUP.md]: Now it is I guess lol

**Comment by S-B99** [comment]
Bump

**Pull request merged**